training/cli.py

```diff
@@ -7,19 +7,26 @@
 import requests
 
 
+class AuthentificationError(Exception):
```

Comment:
I also feel that `AuthenticationError` is a bit too vague considering that users are authenticated via Firebase and developers use AWS SSO to authenticate themselves into AWS. Could we make the naming of the error message class a bit more specific @LuHG18